openupgradelib/openupgrade_tools.py

```diff
@@ -104,6 +104,7 @@ def convert_xml_node(
     style_rm=frozenset(),
     tag="",
     wrap="",
+    attr_rp=None,
```

Comment:
Here i added anaother option to allow attribute replacement, put it in the last to avoid error